spec/src/main/asciidoc/architecture.adoc

```diff
@@ -282,11 +282,19 @@ how such application servers should behave if they want to support MicroProfile
 Metrics from all applications and scopes should be available under a single REST endpoint ending with `/metrics` similarly as
 in case of single-application deployments (microservices).
 
-To help distinguish between metrics pertaining to each deployed application, a tag named `mp_app` should be added to each metric. 
+To help distinguish between metrics pertaining to each deployed application, a tag named `mp_app` should be added to each metric.
```

Comment:
to simplify this a bit, I suggest we fully describe the use of mp_app, on its own, then introduce the problem of metrics that exist in both server and app scopes (combined with need for there to be only one tag set per metrics), which then could lead into the description of `defaultAppName`.  

It would help to follow all of that with the simple rules for a user to follow (if you have multiple apps, use both...)